hooks/pre/30-install_redis_scl.rb

```diff
@@ -0,0 +1,10 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if facts[:os][:release][:major] == '7'
+  package = 'rh-redis5-redis'
+
+  `rpm -q #{package}`
+
+  unless $?.success?
+    execute("yum -y install #{package}")
```

Comment:
I'm wondering if we should just leave this up to Puppet:
```suggestion
    execute("puppet resource package #{package} ensure=installed")
```

Then you can drop the whole `rpm -q` as well.

Reply:
Based on this, I wanted to introduce a single command to do this across the hooks:

https://github.com/theforeman/foreman-installer/pull/440